components/feature/media/src/main/java/mozilla/components/feature/media/service/MediaSessionServiceDelegate.kt

```diff
@@ -163,7 +163,14 @@ internal class MediaSessionServiceDelegate(
         }
     }
 
-    private fun shutdown() {
+    @VisibleForTesting
+    internal fun destroy() {
+        scope?.cancel()
+        audioFocus.abandon()
+    }
+
+    @VisibleForTesting
+    internal fun shutdown() {
         audioFocus.abandon()
```

Comment:
I wonder if we can remove the abandon() call here since we are stopping the service with `stopSelf` and then will do it from onDestroy?

Reply:
Sure, I'll test to confirm.

Reply:
Confirmed that we don't need the `abandon()` call in shutdown anymore.  `stopSelf()` will trigger `onDestroy()` which will in turn call `abandon()`